docs_sphinx/user/nmlimport.rst

```diff
@@ -0,0 +1,322 @@
+NeuroML importer (nmlimport library)
+================
+
+This is a short overview of the `~brian2tools.nmlimport` package, providing
+functionality to import Brian's morphology from a .nml file.
+
+NeuroML is a XML-based description that provides a common data format
+for defining and exchanging descriptions of neuronal cell and network models
+(`NML project website <https://neuroml.org/>`_).
+
+.. contents::
+    Overview
+    :local:
+
+Working example
+---------------
+
+As a demonstration, we will use nmlimport library to generate morphology and
+extract other related information from the ``pyramidal cell's`` nml file. You
+can download it from `here <https://github
+.com/OpenSourceBrain/ACnet2/blob/master/neuroConstruct/generatedNeuroML2/pyr_4_sym.cell.nml>`_.
+
+
+.. code:: python
+
+    from brian2tools.nmlimport.nml import NmlMorphology
+    nml_object = NmlMorphology('pyramidal.cell.nml',name_heuristic=True)
+
+This call provides us the ``nml_object`` that contains all the information
+extracted from `.nml` file. When ``name_heuristic`` param is set to True
```

Comment:
This sentence is a bit long and difficult to understand. Maybe divide it in two?

Reply:
(with "this sentence" I mean: "When ``name_heuristic`` param is set ..."